packages/fiori/src/i18n/messagebundle.properties

```diff
@@ -1,6 +1,71 @@
 #This is the resource bundle for the UI5 Web Components
 #__ldi.translation.uuid=95d47730-48a4-4d6d-92f6-61f8c9d8f274
 
+#XTIT: IllustratedMessage's title for the BeforeSearch illustration
+IM_TITLE_BEFORESEARCH=Let''s get some results
+
+#XTXT: IllustratedMessage's subtitle for the BeforeSearch illustration
+IM_SUBTITLE_BEFORESEARCH=Start by providing your search criteria.
+
+#XTIT: IllustratedMessage's title for the NoActivities illustration
+IM_TITLE_NOACTIVITIES=You''ve not added any activities yet
```

Comment:
You have not added any activities yet

Reply:
Is it expected to have two Apostrophes: "You''ve not"